src/components/tabs/Tabs.vue

```diff
@@ -12,27 +12,59 @@
 </template>
 
 <script>
-  import {createUniqueId} from '../../utils'
-
   export default {
+    data() {
+      return {
+        tabs: []
+      }
+    },
     mounted() {
-      const link = AJS.$(this.$el).find('.menu-item a')[0];
-      const $link = AJS.$(link);
-      AJS.tabs.change($link)
-      AJS.tabs.setup();
+      this.updateTabs()
+      AJS.tabs.setup()
     },
-
-    computed: {
-      tabs() {
-        const tabs = this.$slots.default.filter(slot => slot.componentOptions
-          && (slot.componentOptions.tag === "va-tab" || slot.componentOptions.tag === "aui-tab"));
-        for (let obj of tabs) {
-          if (!obj.tab_id) {
-            obj.tab_id = createUniqueId('tab')
-            obj.componentOptions.propsData.tabId = obj.tab_id
-          }
+    updated(){
+      this.updateTabs()
+    },
+    methods: {
+      selectDefaultTab() {
+        if (this.tabs.length){
+          this.selectTab(this.tabs[0].hash)
         }
+      },
+      hasActiveTab(){
+        let activeTab = $('.menu-item.active-tab')
+        return activeTab.length && activeTab.is(':visible')
+      },
+      selectTab(tabHash) {
+        Vue.nextTick(() => {
+          const tabLinks = this.$refs[`tab_link_${tabHash}`]
+          if (tabLinks && tabLinks.length) {
+            let $tabLink = $(tabLinks[0])
```

Comment:
I believe we should not rely on global jQuery `$` and should switch to `AJS.$`

Reply:
Actually, it breaks on my PC. Same is used above. Another similar thing, you use Vue but don't import it, also breaks.

Reply:
Please check my last commit a6357b2